src/index.js

```diff
@@ -124,6 +124,13 @@ export async function showLocation (options) {
         url += `&from=${sourceLatLng}`
       }
       break
+    case 'truckmap':
+      url = `http://truckmap.com/place/${lat},${lng}`
```

Comment:
Shouldn't this use the `truckmap://` URL scheme?

Reply:
Good question, and thanks for the quick response. The `truckmap://` scheme should allow this library to determine whether or not the app is installed (which the web url can't do), but the app is much better set up to handle the web url scheme. (And Apple [https://developer.apple.com/documentation/uikit/inter-process_communication/allowing_apps_and_websites_to_link_to_your_content](highly recommends) using that instead of the deep link scheme.)

If TruckMap is installed, a url-style universal link is guaranteed to open the app instead of Safari. Is this okay? I can't see any negative repercussions of this, but I'm open to ideas.

Reply:
Sounds okay!